src/js/components/init_define_btn.js

```diff
@@ -1,9 +1,57 @@
 export const initDefineBtn = () => {
-  const defineBtn = document.querySelector('.js-define-btn');
+  const defineBtns = document.querySelectorAll('.js-define-btn');
+  var logo = document.querySelector(".js-shake-logo");
 
-  defineBtn.addEventListener('click', (event) => {
-    document.querySelector('.primary-title').classList.add('expanded');
-    document.querySelector('.definition-list').classList.add('definition-list--active');
-    event.currentTarget.classList.add("define-btn--disabled");
-  });
+  const expandTitle = (title) => {
+    title.classList.add('expanded');
+  }
+
+  const showDefinition = (list) => {
+    list.classList.add('expanded');
+    list.classList.add('definition-list--active');
+  }
+
+  // For init_landing_transition usage
+  const displayLogo = (logo) => {
+    logo.classList.add('js-logo-active');
+  }
+
+  const gsapTransition = () => {
+    const tl = gsap.timeline();
+    var logoY = -logo.getBoundingClientRect().y + logo.getBoundingClientRect().height / 2
+
+    if (logo.classList.contains('js-logo-background')) {
+      TweenLite.set(logo, {y: logoY});
```

Comment:
As a heads up TweenLite is deprecated and we should flip these over to gsap

Reply:
Cool, updated!